.github/scripts/import_experiment.py

```diff
@@ -0,0 +1,248 @@
+    with open(toml_file) as f:
+        toml_data = toml.load(f)
+
+    return toml_data
+
+
+def issue_exists(repo, issue_number):
+    # Verify if the provided issue number is available in the repository
+
+    if not issue_number:
+        return False
+
+    url = f"https://github.com/{repo}/issues/{issue_number}"
+    response = requests.get(url, allow_redirects=False)
+    if response.status_code != 200:
+        return False
+
+    return True
+
+
+def create_issue(repo, experiment_id, file_name, recipe_locales, api_token):
+    # Create a new issue
+
+    url = f"https://api.github.com/repos/{repo}/issues"
+    headers = {"Authorization": f"token {api_token}"}
+
+    issue_body = f"""
+**Experiment ID**: `{experiment_id}`
+**Localization file**: `{file_name}`
+**Locales**: {', '.join(recipe_locales)}
+    """
+    payload = {
+        "title": f"Localization request for experiment {experiment_id}",
+        "body": issue_body,
+    }
+
+    r = requests.post(url=url, headers=headers, data=json.dumps(payload))
+    try:
+        issue_number = r.json()["number"]
+        print(f"Created a new issue: {issue_number}")
+        return issue_number
+    except:
+        print("Error creating a new issue")
+        print(r.status_code)
+        return ""
+
+
+def main():
+    # Read command line input parameters
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--toml", dest="toml_path", help="Path to l10n.toml file", required="True"
+    )
+    parser.add_argument("--id", dest="exp_id", help="Experiment ID", required="True")
+    parser.add_argument("--issue", dest="issue", help="Issue number", default=0)
+    parser.add_argument("--token", dest="token", help="API Token", required=True)
+    parser.add_argument("--repo", dest="repo", help="GitHub repository", required=True)
+    args = parser.parse_args()
+
+    # Store paths
+    script_path = os.path.dirname(__file__)
+    root_path = os.path.abspath(os.path.join(script_path, os.pardir, os.pardir))
+    ftl_path = os.path.join(root_path, "en-US", "subset")
+
+    # Get experiments.json, which contains information about experiments that
+    # were processed through this pipeline.
+    experiments_json = os.path.join(root_path, ".github", "storage", "experiments.json")
+    if not os.path.exists(experiments_json):
+        experiments = {}
+    else:
+        with open(experiments_json) as f:
+            experiments = json.load(f)
+
+    # Get the experiment recipe (JSON), generate the FTL file and store it in
+    # the repository
+    experiment_id = args.exp_id
+    recipe = get_experiment_json(experiment_id)
+    ftl_filename = f"{experiment_id.replace('-', '_')}_{date.today().year}.ftl"
+    with open(os.path.join(ftl_path, ftl_filename), "w") as f:
+        f.write(generate_ftl_file(recipe))
+
+    # Extract the list of locales from the recipe
+    # TODO: https://github.com/mozilla/experimenter/pull/8820
```

Comment:
We might be able to fix this before merging, but this works well enough for now.

The targeting looks something like ``(browserSettings.update.channel == "release") && (version|versionCompare('114.*') <= 0) && (version|versionCompare('113.!') >= 0) && (locale in ['en-US', 'fr', 'it'])``